tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+	"compilerOptions": {
+		
+		"target": "es2016",
+		"module": "commonjs",
+		"esModuleInterop": true,
+		"forceConsistentCasingInFileNames": true,
+		
+    "strict": true,
+		"skipLibCheck": true,
+    "baseUrl": ".",
+    "noEmit": true,
+    "noUnusedParameters": true,
+    "strictNullChecks":true,
+	},
+  "include": [
+    "src"
+  ],
+  "exclude": [
+    "node_modules"
+  ]
```

Comment:
Can we make a the indentations consistent here?

Reply:
Sure, I don't know why I see correct in my editor and here it is not indented. I try to understand it.

Reply:
There's a mix of tab and space indentation here. Your editor might be configured so the width of the tabs makes the file look aligned. But Github uses different tab widths so it shows up differently.